tests/mock_datasource/property_definition.tfmock.hcl

```diff
@@ -0,0 +1,13 @@
+mock_data "opslevel_property_definition" {
+  defaults = {
+    allowed_in_config_files = true
+    description             = "mock-property-definition-description"
+    # id intentionally omitted - will be assigned a random string
```

Comment:
What about `identifier`?

Reply:
We don't need to mock `identifier` since it is the input provided by `data "opslevel_property_definition" "mock_property_definition"` in `tests/data_sources.tf`.
These mock values represent the returned values as if we queried the backend